flax/traverse_util.py

```diff
@@ -159,6 +164,45 @@ def unflatten_dict(xs, sep=None):
     cursor[path[-1]] = value
   return result
 
+def variables_path_partitions(
+    partition_fn: Callable[[Path, Array], Partition]
+) -> Callable[[VariableDict], Dict[Path, Partition]]:
+  """Creates a function that maps a variables structure to a path partitions dictionary 
+  compatible with Optax ``multi_transform`` and ``masked``.
+
+  Here is an example usage with ``multitransform``::
```

Comment:
This module doesn't really have a direct connection to Optax. I think specific examples like this where you combine multiple independent APIs belong in seperate documentation. For example a HOWTO use Optax with Flax.

Reply:
Point taken. It will be featured in the transfer learning tutorial I am creating so maybe that is that. Will modify the docstrings.